src/helpers/discord_utils.ts

```diff
@@ -402,6 +404,7 @@ export async function sendOptionsMessage(messageContext: MessageContext,
             fields,
             footerText,
             thumbnailUrl: KmqImages.LISTENING,
+            color: premiumRequest ? EMBED_SUCCESS_BONUS_COLOR : null,
```

Comment:
This color should be different, and it should also be shown on the "Guess the song" interaction message